DevelopGitHub/GitHubExtension/GitHubExtension.Tools.LocalizationTool/ViewModel/MainWindowViewModel.cs

```diff
@@ -0,0 +1,240 @@
+    {
+        private readonly ITranslationData _translationDataTable;
+
+        private readonly Translator _translator;
+
+        private bool _translateButtonIsEnabled = true;
+
+        private string _translateButtonContent = "Web Translate";
+
+        private Language _sourceLanguage;
+
+        private Language _targetLanguage;
+
+        public MainWindowViewModel()
+        {
+            RegisterCommands();
+
+            _translationDataTable = new TranslationDataTable
+            {
+                TranslationTable = new ObservableCollection<TranslationDataRow>()
+            };
+
+            _translator = new Translator(_translationDataTable);
+
+            //OpenJson();
+        }
+
+        public ICommand CloseCommand { get; set; }
+
+        public ICommand TranslatingButtonCommand { get; set; }
+
+        public ICommand AddEmptyRowCommand { get; set; }
+
+        public ICommand SaveJsonCommand { get; set; }
+
+        public ICommand OpenJsonCommand { get; set; }
+
+        public ICommand ClearTranslationDataCommand { get; set; }
+
+        public ObservableCollection<TranslationDataRow> TranslationData
+        {
+            get
+            {
+                return _translationDataTable.TranslationTable;
+            }
+
+            set
+            {
+                _translationDataTable.TranslationTable = value;
+            }
+        }
+
+        public string TranslateButtonContent
+        {
+            get
+            {
+                return _translateButtonContent;
+            }
+
+            set
+            {
+                _translateButtonContent = value;
+                RaisePropertyChanged(() => TranslateButtonContent);
+            }
+        }
+
+        public bool TranslateButtonIsEnabled
+        {
+            get
+            {
+                return _translateButtonIsEnabled;
+            }
+
+            set
+            {
+                _translateButtonIsEnabled = value;
+                RaisePropertyChanged(() => TranslateButtonIsEnabled);
+            }
+        }
+
+        public bool IsChecked
+        {
+            get;
+            set;
+        }
+
+        public Language SourceLanguage
+        {
+            get
+            {
+                return _sourceLanguage;
+            }
+
+            set
+            {
+                _sourceLanguage = value;
+                RaisePropertyChanged(() => SourceLanguage);
+            }
+        }
+
+        public Language TargetLanguage
+        {
+            get
+            {
+                return _targetLanguage;
+            }
+
+            set
+            {
+                _targetLanguage = value;
+                RaisePropertyChanged(() => TargetLanguage);
+            }
+        }
+
+        public IEnumerable<Language> LanguageEnumValues
+        {
+            get
+            {
+                return Enum.GetValues(typeof(Language))
+                    .Cast<Language>();
```

Comment:
This property executes only twice for all runtime of application. 
I think I don't need converter in this case.
What you think?